### ISSUE TITLE
Why we can't have access to more colors?

### ISSUE THREAD

**Issue by marieeved** [wet-boew/wet-boew#8292]
Hi,

Within the WET code, we can use only the "[alert color](http://wet-boew.github.io/wet-boew-styleguide/v4/design/colour-en.html)" and all the inline style is erased automatically by Teamsite.

I know that it's written that it's because of accessibility issue, but I think that if we used also text with the color or invisble text for accessibility, in my head, it should be ok to use more colors for a website (same as with images)..

On day-to-day, a colorful website (with a lot of images) that's what people are used to see and that's what people want too.

Can someone explain me why we can't use more colors on our websites or if there is a way to do it, tell me how?

I work with an intranet and IE11.

Thanks a lot,

**Comment by duboisp** [comment]
@marieeved The use of those color "[alert color](http://wet-boew.github.io/wet-boew-styleguide/v4/design/colour-en.html)" is more based on the purpose rather than the actual color.

This project might interest you and they would welcome any additional help: https://github.com/gctools-outilsgc/design-system Eventually, when their work would be more in a mature state, it would cross the path of the wet-boew project.

**Comment by duboisp** [comment]
> all the inline style is erased automatically by Teamsite.

You should send a request to your internal Teamsite support team.

**Comment by marieeved** [comment]
@duboisp 
So, it's not supposed to do that? 
I thought that it was for limit different style/design between websites (to create uniformity)...

**Comment by duboisp** [comment]
> So, it's not supposed to do that?
I assume you refer to my comments regarding the removal of inline style. WET has nothing to do with that and there is no code that auto-remove any inline styling. For me, that is more a design decision from your web content management system team.

> I thought that it was for limit different style/design between websites (to create uniformity)...

This is limited to the theme style only, not globally. Although the baseline CSS reuse in every theme include a default style. Then it is up to the theme to overwrite those style. If none do, then yes we can have the impression that WET enforce an uniformity, but in reality that is still a theme related decision.

Regarding the intranet theme, I know it's implementation may vary from institution/organization compared to each others.
